java/java-common/src/main/java/org/cloudburstmc/protocol/java/packet/play/clientbound/SetPlayerTeamPacket.java

```diff
@@ -76,4 +75,35 @@ public static NameTagVisibility getById(int id) {
             return VALUES.length > id ? VALUES[id] : null;
         }
     }
+
+    public enum TeamColor {
```

Comment:
This can be entirely replaced with `NamedTextColor` and `TextDecoration` inside of adventure.